src/openMVG/matching_image_collection/gpu/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+project(openMVG_gpu_matching_image_collection)
+
+if (OpenMVG_USE_CUDA)
```

Comment:
Ah, makes sense. Odd: when using the totally uppercase version, the system seems to ignore the clause